sunpycube/cube/datacube.py

```diff
@@ -582,11 +582,12 @@ def _new_instance(cls, data_list, meta=None, common_axis=None):
         """
         return cls(data_list, meta=meta, common_axis=common_axis)
 
-    def index_as_cube(self, item):
+    @property
+    def index_as_cube(self):
```

Comment:
Could you include an example in this docstring of how this function should be used?